docs/pages/_app.tsx

```diff
@@ -95,7 +95,20 @@ export default function MyApp({ Component, pageProps }: AppProps) {
             inset: 0;
             top: var(--top-nav-height);
             background: var(--primary-bg-color);
-            display: ${navOpen ? "block" : "none"};
+            z-index: 100;
+
+            transition: opacity 0.2s, transform 0.2s;
```

Comment:
Can we use our transition variables that we have defined:

```
--transition: all var(--transition-duration) var(--transition-fn);
```

Or some other variable we have